SpiceCompanion/Spice.entitlements

```diff
@@ -2,8 +2,13 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:example.com</string>
```

Comment:
This is unused by the new code and can be removed right?

Reply:
Yes, also I forgot to remove the ".beta" in the Bundle Identifier.